extensions/wikia/Rail/styles/RailElement.scss

```diff
@@ -0,0 +1,264 @@
+@import "skins/shared/color";
+
+$black: #000;
+$white: #FFF;
+$color-divider: #E6EBF2;
+$color-rail-background: mix($color-page, $black, 90%);
+@if $is-dark-wiki {
+    $color-rail-background: mix($color-page, $white, 90%);
+}
+
+@mixin standard-primary-button() {
+    background-image: none;
+    border: 0;
+    border-radius: 0;
+    color: $white;
+    text-transform: uppercase;
+
+        &:before {
+            display: none;
+        }
+}
+
+@mixin wikia-rail-element() {
+    background-color: $color-rail-background;
+    margin-bottom: 10px;
+    padding: 20px;
+    position: relative;
+    word-wrap: break-word;
+}
+
+.wikia-rail {
+    float: right;
+    overflow: hidden;
+    padding: 0 10px 10px 0;
+    width: 300px;
+
+    h1 {
+        padding-left: 0;
+    }
+
+    .WikiaActivityModule {
+        @include wikia-rail-element;
+        font-size: 12px;
+        font-weight: bold;
+
+        a {
+            color: $color-links-default;
+            
+            &:hover {
+                color: $color-links-default;
+                text-decoration: underline;
+                text-shadow: initial;
+            }
+        }
+
+        em a {
+            font-size: 16px;
+            font-weight: bold;
+            margin-top: 20px;
+        }
+
+        .activity-heading {
+            font-size: 24px;
+            font-weight: bold;
+            margin-bottom: 20px;
+        }
+
+        .edited-by {
+            font-size: 12px;
+            font-weight: normal;
+            margin-top: 10px;
+
+            a {
+                font-size: 12px;
+            }
+        }
+
+        .more {
+            float: right;
+            font-size: 12px;
+            font-weight: bold;
+            margin-top: 30px;
+            text-transform: uppercase;
+        }
+
+        .sprite {
+            left: 3px;
+            position: absolute;
+            top: 8px;
+        }
+    }
+
+    .videos-module-rail {
+        @include wikia-rail-element;
+        display: block;
+
+        h1 {
+            font-size: 24px;
+            font-weight: bold;
+            margin-bottom: 16px;
+        }
+
+        .thumbnails {
+
+            a:first-of-type {
+                color: $color-links;
+                margin-bottom: 0;
+
+                &:hover {
+                    color: $color-links;
+                    text-decoration: underline;
+                    text-shadow: initial;
+                }
+            }
+
+            .title {
+                font-size: 12px;
+                height: 40px;
+                margin-top: 10px;
+
+                a {
+                    display: block;
+                    margin: 0;
+                    overflow: hidden;
+                    text-overflow: ellipsis;
+                    white-space: nowrap;
+                    word-wrap: normal;
+                }
+            }
+        }
+    }
+
+    &.new-chat {
```

Comment:
some parts of chat module are still styled based on old styles (.ChatModule .chat-contents and .ChatModule .chat-joins)